src/main/external-resources/renderers/BlackBerry-PlayBook-KalemSoftMP.conf

```diff
@@ -39,7 +39,7 @@ Supported = f:wmv                 m:video/x-ms-wmv
 Supported = f:wav    m:audio/wav        n:6   a:dts|lpcm     s:48000
 Supported = f:wav    m:audio/wav        n:2                  s:48000
 Supported = f:mp3    m:audio/mpeg       n:2
-Supported = f:aac    m:audio/x-m4a      n:2   a:(?!alac).+
+Supported = f:m4a    m:audio/x-m4a      n:2   a:(?!alac).+
```

Comment:
@Sami32 If you know of an error, please try to explain it instead of just giving up. It's not that I don't *want* to understand, I simply don't understand what you mean.